### PR TITLE
Virtual devices description improved in internet connection test client

### PR DESCRIPTION
**Pull request by mchf**
bnc#913722

### CODE REVIEW

**Comment by mvidner** [comment]
Please add the comments, code LGTM.

**Comment by mchf** [comment]
this code is dead in master at least partly ;-) I need to walk trough this but at least the "Internet ..." client was excluded from installation workflow in the new installer.

File: src/routines/widgets.ycp [hunk 1]
```diff
@@ -243,29 +243,44 @@ map<string, any> mtu_widget()
...
 
-define string GetDeviceDescription(string device_id) {
-    string device_name = NetworkInterfaces::GetValue(device_id,"NAME");
-    if ( (device_name == nil) || (device_name == "") ) {
-        //TRANSLATORS: Informs that device name is not known
-        device_name = _("Unknown device");
-    }
+// Provides human readable device description
+// @param device_id system wide device name (e.g. eth0)
+// @return hwname, or if it is empty, a translated generic name based on devtype
```
Comment (mvidner) [inline src/routines/widgets.ycp]: Sorry, I meant this wording to be applied to `CheckEmptyName`, where it is really needed because of the confusing name.

**Comment by mvidner** [comment]
> \# LGTM

**Comment by mchf** [comment]
Thanks

**Pull request merged**